packages/MarkdownEditor-Tests.package/MarkdownHeadingsTest.class/instance/testAsTextStylesInline.st

```diff
@@ -0,0 +1,12 @@
+tests
+testAsTextStylesInline
```

Comment:
I hope this method is understandable. I tried to make clear that the mock inline parser just returns itself to check if the parser is used with the expected inline content, but if this is not clear, we'll think of something else.

Reply:
I really don't think this is understandable. I don't like how we are using the MockObject. It is not clear why `assert: inlineParserMock identical: result` should be the case.

Also It needs to be tested that parsing the inline content doesn't remove the Heading Styling. But maybe that should be in a different test for the inline parser.

Reply:
Yep, I agree, I don't see what you're trying to do just from reading the code. And maybe the name could be something like 'testConvertingHeadingToTextAppliesInlineStyles' or something like this, if that is what it does